docs/src/main/sphinx/installation/cli.rst

```diff
@@ -381,10 +383,39 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Batch mode
+----------
+
+Running the Trino CLI with the ``--execute``, ``--file`` or passing queries to
+the standard input uses the batch mode (non-interactive), in which
+the Trino CLI exits after executing all queries, does not report progress,
+and prints results in the ``CSV`` format.
+
+The following options are available to further configure the CLI in batch
+mode:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--execute=<execute>``
+    - Execute specified statements and exit.
+  * - ``-f``, ``--file=<file>``
+    - Execute statements from file and exit.
+  * - ``--ignore-errors``
+    - Continue processing in batch mode when an error occurs (default is to
+      exit immediately).
+  * - ``--output-format=<format>``
+    - :ref:`cli-output-format` for batch mode.
+  * - ``--progress``
+    - Show query progress in batch mode.
```

Comment:
Clarify that this does NOT affect the output .. at least I assume so .. also how does that work when piping into a file

Reply:
Yeah, looks like it's printed to stderr. Lines updates are somehow messed up when redirecting to a file, no matter if there's a TTY or not (piping vs --execute):
```

jwas@janmac ~/src/trino (git)-[cli-docs-update] % trino --server localhost:8080 --user trino-client --catalog hive --schema v2 --execute 'select count(*) from tpch.sf100.orders;' --progress
"150000000"

Query 20220422_091355_00004_mit6n, FINISHED, 1 node
Splits: 21 total, 21 done (100.00%)
59.08 [150M rows, 0B] [2.54M rows/s, 0B/s]

jwas@janmac ~/src/trino (git)-[cli-docs-update] % trino --server localhost:8080 --user trino-client --catalog hive --schema v2 --execute 'select count(*) from tpch.sf100.orders;' --progress > out.txt
Apr 22, 2022 11:15:02 AM org.jline.utils.Log logr
WARNING: Unable to create a system terminal, creating a dumb terminal (enable debug logging for more information)
Query 20220422_091502_00005_mit6n [RUNNING] i[0 0B 0B] o[0 0B 0B] splits[0/21/0]
Query 20220422_091502_00005_mit6n, FINISHED, 1 node
Splits: 21 total, 21 done (100.00%)
51.28 [150M rows, 0B] [2.93M rows/s, 0B/s]

jwas@janmac ~/src/trino (git)-[cli-docs-update] % cat out.txt 
"150000000"
jwas@janmac ~/src/trino (git)-[cli-docs-update] % echo 'select count(*) from tpch.sf100.orders;' | trino --server localhost:8080 --user trino-client --catalog hive --schema v2 --progress                              
"150000000"422_091618_00006_mit6n [RUNNING] i[75M 0B 0B] o[75M 0B 0B] splits[0/19/2]

Query 20220422_091618_00006_mit6n, FINISHED, 1 node
Splits: 21 total, 21 done (100.00%)
1:01 [150M rows, 0B] [2.47M rows/s, 0B/s]
```
Important thing is that it does not affect the output.

Reply:
Lets add that important fact..

Reply:
I did, see the latest comment. I'll mark this as resolved since it's attached at the old diff.